src/routines/generic_templates.md

```diff
@@ -0,0 +1,23 @@
+# Generic Subroutine Templates
```

Comment:
Is this necessary?

Reply:
Probably right, this kind of thing can just stay local to me while the subroutines are generated